src/rnp/fficli.cpp

```diff
@@ -2692,9 +2692,19 @@ cli_rnp_encrypt_and_sign(const rnp_cfg &cfg,
     if (cfg.get_bool(CFG_ENCRYPT_SK)) {
         std::string halg = cfg.get_hashalg();
         std::string ealg = cfg.get_str(CFG_CIPHER);
+        size_t      iterations = cfg.get_int(CFG_S2K_ITER);
+
+        if (!iterations) {
+            if (rnp_calculate_iterations(
```

Comment:
We should call `rnp_calculate_iterations()` only if user passed `--s2k-msec` option, as otherwise default value would be used, without need of extra API call.

Reply:
fixed